test.ps1

```diff
@@ -26,21 +26,30 @@
     .NOTES
         Tests are excluded with Tags VersionChecks, StyleRules, Help.
 #>
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSUseDeclaredVarsMoreThanAssignments", "ErrorView")]
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSUseDeclaredVarsMoreThanAssignments", "PesterPreference")]
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSAvoidUsingWriteHost", "")]
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute("Pester.BuildAnalyzerRules\Measure-SafeCommands", "")]
 param (
     # force P to fail when I leave `dt` in the tests
+    [Parameter(Mandatory=$false)]
     [switch] $CI,
+    [Parameter(Mandatory=$false)]
     [switch] $SkipPTests,
+    [Parameter(Mandatory=$false)]
     [switch] $NoBuild,
+    [Parameter(Mandatory=$false)]
```

Comment:
Are these changes necessary? Did any PSSA rule warn on them? `[Parameter()]` makes this an advanced function, so `$args` won't be supported which is why the CI failed.

If these Parameter-attributes should stay then the unused `-Passthru` argument below needs to go. https://github.com/pester/Pester/blob/148d39acd7714b0dd6ca1dedbf84914106a39e51/azure-pipelines.yml#L121

Reply:
Not necessary and removed.